eth_utils/conversions.py

```diff
@@ -90,7 +99,7 @@ def to_bytes(primitive=None, hexstr=None, text=None):
         return to_bytes(hexstr=to_hex(primitive))
     elif hexstr is not None:
         if len(hexstr) % 2:
-            hexstr = '0x0' + remove_0x_prefix(hexstr)
+            hexstr = '0x0' + remove_0x_prefix(hexstr)  # type: ignore
```

Comment:
Generally good to have an additional comment explaining why we have to `# type: ignore` (normally on the preceding line).